lib/plugins/wallet/cardano/cardano.js

```diff
@@ -0,0 +1,85 @@
+const { walletId } = require('./mnemonic')
+const cardanoApi = require('./client')
+
+// TODO: work with BN here instead of api.js?
+
+async function checkCryptoCode(cryptoCode) {
```

Comment:
does this need to be `async` at all?

Reply:
i copy this from other currencies, they always create Promise chain, so from this point of view i suppose that's make sense